svo (curing skeleton, controllers, action system).xml

```diff
@@ -2333,6 +2333,19 @@ function svo.sk.fix_affs_and_defs()
     svo.addaffdict(svo.dict.deafaff)
     svo.echof("deafness is now considered an affliction, will cure it.")
   end
+  
+    --ablaze check for fire elementals, they always must be ON FIRE!
+  if affs.ablaze and ((defdefup[defs.mode].torch) or (conf.keepup and defkeepup[defs.mode].torch)
+    and (svo.me.class == 'fire Elemental Lord' or svo.me.class == 'fire Elemental Lady')) then
+    svo.rmaff('ablaze')
+    defences.got('torch')
+    svo.echof("ablaze is now considered a defence.")
+  elseif defc.torch and not ((defdefup[defs.mode].torch) or (conf.keepup and defkeepup[defs.mode].torch)
+   and (svo.me.class ~= 'fire Elemental Lord' or svo.me.class ~= 'fire Elemental Lady')) then
```

Comment:
This should be an and, not an or, or it will always be a true statement. When you are a fire Lord, it is true that you are not a fire Lady